src/core/CallTracker.js

```diff
@@ -125,6 +125,24 @@ getJasmineRequireObj().CallTracker = function(j$) {
     this.saveArgumentsByValue = function() {
       opts.cloneArgs = true;
     };
+
+    /**
```

Comment:
I don't think markInteractionAsChecked and getInteractionChecked should be part of Jasmine's public API, so they shouldn't have jsdocs.